docs/xdr/features/collect/integrations/network/sonicwall_sma.md

```diff
@@ -0,0 +1,37 @@
+uuid: 622999fe-d383-4d41-9f2d-eed5013fe463
+name: SonicWall SMA
+type: intake
+
+## Overview
+
+SonicWall Secure Mobile Access offers secure and seamless remote access to corporate resources, applications, and data, enhancing workforce mobility while maintaining robust security and compliance measures.
+
+{!_shared_content/operations_center/detection/generated/suggested_rules_622999fe-d383-4d41-9f2d-eed5013fe463_do_not_edit_manually.md!}
+
+{!_shared_content/operations_center/integrations/generated/622999fe-d383-4d41-9f2d-eed5013fe463.md!}
+
+## Configure
+
+This setup guide will show you how to forward your SonicWall SMA logs to Sekoia.io by means of a syslog transport channel.
+
+### Prerequisites
+
+- Must have GMS server or On-Prem Analytics server installed and configured.
+- Have an Address Object Created on the Firewall for SonicWall Analytics system.
```

Comment:
Where do you get this information?

Reply:
Yes my fail. Did not cleanup from `sonicwall fw`. Thank you!